### DISCUSSION TITLE
Error completing request when generating video

### DISCUSSION THREAD

**Discussion by vitalbuttering** [thygate/stable-diffusion-webui-depthmap-script#354] [General]
Anyone can diagnose why i am getting this error when i try to generate video on Depth?

Error completing request
Arguments: (None, '300', '40', 1, '-0.015, 0.0, -0.05', '0.03, 0.03, 0.05, 0.03', False, 'mp4', '3') {}
Traceback (most recent call last):
  File "I:\stable-diffusion-webui\modules\call_queue.py", line 56, in f
    res = list(func(*args, **kwargs))
  File "I:\stable-diffusion-webui\modules\call_queue.py", line 37, in f
    res = func(*args, **kwargs)
  File "I:\stable-diffusion-webui\extensions\stable-diffusion-webui-depthmap-script\src\core.py", line 590, in run_makevideo
    if len(fn_mesh) == 0 or not os.path.exists(fn_mesh):
TypeError: object of type 'NoneType' has no len()

**Comment by graemeniedermayer** [comment]
This feels like it must be a gradio library issue (the ui library). It might be helpful to know what version of gradio your running. You can find that out with:

Linux:
```pip freeze | grep gradio```

Windows:
```pip freeze | findstr gradio```

**Reply by vitalbuttering** [reply]
Figured it out. I needed to tick Generate 3D mesh. Anyway is there a way to fade in and fade out looping videos generated within this extension?

**Reply by graemeniedermayer** [reply]
> fade in and fade out

Do you mean speed wise or like a blackout fade? Either way they would require a little bit of extra coding. 

The setting are a quite simplified right now, but the underlying algorithm is quite powerful and can do more than the setting imply.

**Reply by vitalbuttering** [reply]
More like a transition fade rather than a blackout fade. But yeah an alternative is to just port it to Premiere and do it there.
